tokens/utu.json

```diff
@@ -0,0 +1,10 @@
+    "aurora_address": "0x7aa1a72f744cdcd4c89918aebfbe4f1d1d1156e6",
+    "name": "UTU Coin",
+    "symbol": "UTU",
+    "decimals": 18,
+    "data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMzQuNDMgMTM0LjQzIj48ZGVmcz48c3R5bGU+LmNscy0xe2ZpbGw6I2ZmZjt9LmNscy0yLC5jbHMtM3tmaWxsLXJ1bGU6ZXZlbm9kZDt9LmNscy0ze2ZpbGw6I2ZkMzt9PC9zdHlsZT48L2RlZnM+PGcgaWQ9IkxheWVyXzIiIGRhdGEtbmFtZT0iTGF5ZXIgMiI+PGcgaWQ9IkxheWVyXzEtMiIgZGF0YS1uYW1lPSJMYXllciAxIj48Y2lyY2xlIGNsYXNzPSJjbHMtMSIgY3g9IjY3LjIxIiBjeT0iNjcuMjEiIHI9IjY3LjIxIi8+PHBhdGggY2xhc3M9ImNscy0yIiBkPSJNMTA5LjQ3LDU2LjI0YTcuNTksNy41OSwwLDAsMC02LjA4LDQuNjdBNy4zLDcuMywwLDAsMCwxMDMsNjVsLTkuOSwyLjc3LjQ5LDEuNzEsOS45My0yLjc4YTcuNTMsNy41MywwLDEsMCw2LTEwLjQ5WiIvPjxwYXRoIGNsYXNzPSJjbHMtMiIgZD0iTTEwOS4xMiw3Ni4yN2E2LjU0LDYuNTQsMCwwLDAtNi44NSw0LjExTDk0LDc5LjA4bC0uMjcsMS43Niw4LjExLDEuMjhBNiw2LDAsMCwwLDEwMy42NCw4N2E2LjQyLDYuNDIsMCwwLDAsNC40NSwyLjI0LDYuNTEsNi41MSwwLDAsMCwxLTEzWiIvPjxwYXRoIGNsYXNzPSJjbHMtMiIgZD0iTTEwMC4xLDkyLjRBNS40Niw1LjQ2LDAsMCwwLDk2LDk2LjQ4LDUuNTYsNS41NiwwLDAsMCw5Ni4zLDEwMGwtNC43OCwzLjlhNi40OCw2LjQ4LDAsMCwwLTUuMTYtMi42M0w4NCw5Mi4zOGwtMS43Mi40NiwyLjMxLDguNjlhNi41MSw2LjUxLDAsMCwwLTQuNyw1bC02LjE2LS4yNEExMS41NSwxMS41NSwwLDAsMCw2Niw5NS45M2wzLTUuNTEtMS41OC0uODQtMy4xNiw1Ljg5YTExLjE4LDExLjE4LDAsMCwwLTYuMjUuNjQsMTEuNTIsMTEuNTIsMCwxLDAsMTUuNjgsMTJsNi4xMi4yNGE2LjUyLDYuNTIsMCwxLDAsMTIuNjEtMi44M2w0Ljg5LTRhNS41Miw1LjUyLDAsMSwwLDIuODItOS4xM1oiLz48cGF0aCBjbGFzcz0iY2xzLTIiIGQ9Ik00OC41Nyw4Ny43MmE2LjU5LDYuNTksMCwwLDAsNC4xNS4zNSw2LjUxLDYuNTEsMCwwLDAsMS0xMi4zMWwuNjYtOC41OGMuMjUsMCwuNTEsMCwuNzcsMEE4LjUsOC41LDAsMCwwLDYxLjg4LDY0bDYuNDQsNCwuOTQtMS41Mi02LjQzLTRhOC40Miw4LjQyLDAsMCwwLC44OC0zLjc1LDguNTMsOC41MywwLDAsMC02LjktOC4zNkw2MCw0MS44NmExMywxMywwLDAsMCwxMC43Ni0yLjQ3bDQuODIsM0E4LjUyLDguNTIsMCwwLDAsODAuMzksNTZsLTEsOC41LDEuNzcuMiwxLTguNDloLjFBOC41Myw4LjUzLDAsMCwwLDg2LDQwbDEuODUtNS4zNGE2LjU3LDYuNTcsMCwwLDAsNS44Ny0uNDdsNC4zOSwzLjc3YTguNDUsOC40NSwwLDEsMCwxLjU5LTFMOTUuMSwzM2E2LjQ4LDYuNDgsMCwwLDAsMS43MS00Ljc2LDYuNTIsNi41MiwwLDAsMC0xMi41NS0yLjEzbC04LjgzLjE2YTEzLDEzLDAsMSwwLTE3LjQsMTVsLTMuMzEsOC44OWE4LjU0LDguNTQsMCwwLDAtOC4wOCw4LjUxLDcuOTQsNy45NCwwLDAsMCwwLC44NUwzOS42LDYxLjg3YTEzLDEzLDAsMCwwLTEwLTQuNjhoLS4yNkwyOC43Nyw1MmE2LjUxLDYuNTEsMCwwLDAsNC43Ni01LjQ1bDQuMTMtLjA5YTUuNTIsNS41MiwwLDEsMCw1LjgyLTYuMjZMNDMsMzUuNzlhNS41Myw1LjUzLDAsMSwwLTIsLjMzbC40Niw0LjI2YTUuNTMsNS41MywwLDAsMC0zLjc0LDRsLTQuMjcuMDlhNi41Miw2LjUyLDAsMSwwLTYuNzEsNy42N2wuNTYsNS4yMWExMywxMywwLDEsMCwxMy40OCw2LjJsNi4zLTIuMDhhOC41OCw4LjU4LDAsMCwwLDUuMzEsNS4yNmwtLjY2LDguNWMtMS43OC0uMTYtMy43NC4yOS02LjA4LDMuMzdhNSw1LDAsMCwwLTEsMi43MUE2LjQ5LDYuNDksMCwwLDAsNDcsODYuNzZsLTMuNzEsMy41YTksOSwwLDEsMCwxLjI3LDEuMjVabTM1LjI0LTU5LjhjMCwuMjIsMCwuNDUsMCwuNjlhNi41LDYuNSwwLDAsMCwyLjQ5LDUuMTFsLTIsNS42NmE4LjUsOC41LDAsMCwwLTcuNSwxLjczbC00LjcxLTNhMTMsMTMsMCwwLDAsMy42NS05YzAtLjM2LDAtLjcsMC0xLjA1WiIvPjxwYXRoIGNsYXNzPSJjbHMtMyIgZD0iTTc5Ljc2LDYwLjJhMTguMDUsMTguMDUsMCwxLDEtMTguMDYsMTgsMTguMDYsMTguMDYsMCwwLDEsMTguMDYtMThaIi8+PC9nPjwvZz48L3N2Zz4=",
```

Comment:
There are missing fields eg  "near_address" & "icon"

Reply:
@databu will you update this one?

Reply:
Hi, yes, sorry, it slipped my attention. I'm afk this week and next, will update after.

Reply:
Hi, yes, sorry, it slipped my attention. I'm afk this week and next, will update after.

Reply:
Hi, yes, sorry, it slipped my attention. I'm afk this week and next, will update after.